src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue18668.cs

```diff
@@ -0,0 +1,24 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue18668 : _IssuesUITest
+	{
+		public override string Issue => "Visual state change for disabled RadioButton";
+
+		public Issue18668(TestDevice device) : base(device){ }
+
+		[Test]
+		[Category(UITestCategories.RadioButton)]
+		public void TestIssue18668()
+		{
+			App.WaitForElement("button");
+			App.Click("button");
+
```

Comment:
Could the test change the theme and verify the disabled RadioButton on Light and Dark theme?

Reply:
I'm not sure if this is necessary as changing the opacity is rather independent on user theme right?

<img width="396" alt="Screenshot 2024-11-14 at 19 48 58" src="https://github.com/user-attachments/assets/7aaf4e2a-1ef5-46e8-9f07-9bdf5d15e405">